sdk/go/common/testing/environment.go

```diff
@@ -226,17 +228,29 @@ func (e *Environment) SetupCommandIn(dir string, command string, args ...string)
 		passphrase = e.Passphrase
 	}
 
-	if e.UseLocalPulumiBuild && command == "pulumi" {
-		pulumiRoot := os.Getenv("PULUMI_ROOT")
-		if pulumiRoot == "" {
-			home, err := os.UserHomeDir()
+	if command == "pulumi" {
+		if pulumiPath, isSet := os.LookupEnv("PULUMI_INTEGRATION_PATH"); isSet {
+			command = pulumiPath
+		} else if e.UseLocalPulumiBuild {
```

Comment:
Maybe I'm blind, but I don't see this being set to true anywhere? Where is this being set?

Reply:
Correct - from what I can tell it was designed to be added to a specific test when you want to run it locally to make it use the correct binary. In CI, the binary is always at `bin/pulumi`.